upp-elasticsearch-provisioner/cloudformation/upp-ccf-content.yml

```diff
@@ -0,0 +1,70 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: >
+  Template to spin-up an Elasticsearch cluster that contains FT content used by CCF platform
+
+Parameters:
+  EnvironmentType:
+    Description: Environment type (Test or Production).
+    Type: String
+    AllowedValues:
+      - 'd'
+      - 't'
+      - 'p'
+    Default: t
+  ClusterName:
+    Description: The name of the ES cluster
+    Type: String
+  Region:
+    Description: AWS region - only used to generate the domain access policy
+    Type: String
+    AllowedValues:
+      - 'eu-west-1'
+      - 'us-east-1'
+    Default: eu-west-1
+
+Resources:
+  ElasticsearchDomain:
+    Type: "AWS::Elasticsearch::Domain"
+    Properties:
+      DomainName: !Ref ClusterName
+      ElasticsearchVersion: "7.4"
+      ElasticsearchClusterConfig:
+        InstanceCount: "3"
+        InstanceType: "m5.xlarge.elasticsearch"
+        DedicatedMasterEnabled: "true"
+        DedicatedMasterType: "m5.large.elasticsearch"
+        DedicatedMasterCount: "3"
+      EBSOptions:
+        EBSEnabled: true
+        Iops: 0
+        VolumeSize: 30
+        VolumeType: "gp2"
+      SnapshotOptions:
+        AutomatedSnapshotStartHour: "0"
+      AccessPolicies:
+        Version: "2012-10-17"
+        Statement:
+          -
+            Effect: "Allow"
+            Principal:
+              AWS: !Join ["", ["arn:aws:iam::", !Ref 'AWS::AccountId', ":user/content-containers-apps"]]
+            Action: "es:*"
+            Resource: !Join [ "", [ "arn:aws:es:", !Ref Region, ":", !Ref 'AWS::AccountId', ":domain/", !Ref ClusterName, "/*" ] ]
+      AdvancedOptions:
+        rest.action.multi.allow_explicit_index: "true"
+      Tags:
+        - Key: environment
+          Value: !Ref EnvironmentType
+        - Key: teamDL
+          Value: "universal.publishing.platform@ft.com"
+        - Key: systemCode
+          Value: "upp"
+        - Key: ipcode
```

Comment:
I've not seen this code before. Do we need it?

Reply:
I talked with Platform Health and it looks like this one is deprecated so I'm removing it.